pkg/startup/startup.go

```diff
@@ -1142,3 +1187,48 @@ func terminate() {
 	log.Warn("Terminating")
 	exitFunction(1)
 }
+
+// extractKubeadmCIDRs looks through the config map and parses lines starting with 'podSubnet'.
+func extractKubeadmCIDRs(kubeadmConfig *v1.ConfigMap) (string, string, error) {
+	var v4, v6 string
+	var line []string
+	var err error
+
+	if kubeadmConfig == nil {
```

Comment:
FWIW, it looks like we already check this prior to calling this function, so we probably don't need this check here. 

It's not wrong, per-se, but makes it less clear who is responsible for checking whether the configmap exists or not.

Reply:
Yes.. If this was in C I would have put an assertion there :-) This is me trying to write self-documenting code so if anyone uses this routine in the future they'll know this argument can't be null.

Reply:
lol gotcha.